scripts/py_matter_idl/matter_idl/generators/kotlin/MatterClusters.jinja

```diff
@@ -58,8 +123,19 @@
 
 package matter.devicecontroller.cluster.clusters
 
-import matter.controller.MatterController
+import java.util.logging.Level
+import java.util.logging.Logger
+import java.time.Duration
+import matter.controller.*
```

Comment:
can we avoid star imports or are these actually dynamic?

Reply:
Yes, I have updated the package name for core class to 'matter.controller' based on the review comment, but the cluster lib is still under package 'matter.devicecontroller'.

In the follow-up PR, I will change the package of cluster lib to package matter.controller.cluster.clusters,  it becomes a part of the matter.controller hierarchy, then we will not need the import matter.controller.* statement anymore.